movielist/src/components/features/shows/ReviewForm.tsx

```diff
@@ -1,42 +1,65 @@
-import { Textarea, Button, Input, Flex, RadioGroup, Radio } from "@chakra-ui/react";
+import { Textarea, Button, Input, Flex, RadioGroup, Radio, chakra, FormControl } from "@chakra-ui/react";
 import { useState } from "react";
 import { IReviewItem } from "../reviews/ReviewItem";
+import { useForm } from "react-hook-form";
+import useSWRMutation from "swr/mutation";
+import { mutator, postReview } from "@/fetchers/mutators";
+import { useParams } from "next/navigation";
 
 interface IReviewAddForm {
     onAdd: (review: IReviewItem) => void;
 }
 
+export interface IReviewInputs {
+    comment: string,
+    rating: number,
+    show_id: number
+}
 export default function ReviewForm({onAdd}: IReviewAddForm) {
+    const params = useParams();
+    const {register, handleSubmit} = useForm<IReviewInputs>();
     const [score, setScore] = useState('1');
     const [comment, setComment] = useState('');
-    const onClickHandler = () => {
+    const {trigger} = useSWRMutation("https://tv-shows.infinum.academy/reviews",postReview);
+    const onClickHandler = async (data: IReviewInputs) => {
+        data.rating = parseInt(score);
+        await trigger(data).then(() => {
+            window.location.reload();
+        });
```

Comment:
Single page aplikacije radimo baš radi toga što želimo izbjeći reloadanje stranice.
Podatke treba updateati swr mutacijama (onSucces)

Reply:
prebacio updateanje reviewova na swrmutation